proofs/signatures/th_lira.plf

```diff
@@ -353,3 +353,49 @@
     (! is_affy (is_aff y aff_y)
       (! a (^ (aff_mul_c aff_y x) aff_z)
         (is_aff (*_Real y (a_real x)) aff_z))))))))
+
+;; ========================== ;;
+;; Bounds on Affine Functions ;;
+;; ========================== ;;
+
+; Bounds that an affine function might satisfy
+(declare bound type)
+(declare bound_pos bound)           ; > 0
+(declare bound_non_neg bound)       ; >= 0
+
+; formulas over affine functions
+(declare bounded_aff (! a aff (! b bound formula)))
```

Comment:
Maybe document something like: Given `a` and `b`, what does `f` represent?

Reply:
Good idea. Will do!